plasmapy/formulary/braginskii.py

```diff
@@ -482,9 +493,11 @@ def resistivity(self) -> u.Ohm * u.m:
 
     @property
     def thermoelectric_conductivity(self):
-        """
+        r"""
         Calculate the thermoelectric conductivity.
 
+        The thermoelectric conductivity (:math:`\hat{\beta}`) of a plasma is defined by...
```

Comment:
Looks like this one's missing a continuation? I'd be fine merging it as is, since this is, of course, an incremental improvement over the previous state of this module.

Maybe a [TODO note](https://www.sphinx-doc.org/en/master/usage/extensions/todo.html#directive-todo) would be useful, for easier text search of these!

Reply:
Yes, I just left off with "..." because I could not parse how the method works with the multiple models supported. But I added todo's for picking up on these formulas later on.

Reply:
The todo extension is not in our conf.py file for the docs. Should I add it there in this PR, or do you think #TODO is fine? The todo directive would not be displayed in the build, but #TODO would be I believe.

Reply:
Completely forgot we didn't have that in our config! Added it, let's see if it works :)